src/main/java/appeng/client/render/FacadeDispatcherBakedModel.java

```diff
@@ -21,7 +21,6 @@
 
 import java.util.Collections;
 import java.util.List;
-
 import javax.annotation.Nullable;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 'javax.annotation.Nullable' should be separated from previous imports. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)